openmdao/drivers/tests/test_scipy_optimizer.py

```diff
@@ -115,7 +115,7 @@ def test_design_vars_on_all_procs_scipy(self):
 
         prob.set_solver_print(level=0)
 
-        prob.driver = om.ScipyOptimizeDriver(optimizer='SLSQP', tol=1e-6, disp=False)
+        prob.driver = om.ScipyOptimizeDriver(optimizer='SLSQP', tol=1e-5, disp=False)
```

Comment:
is this tolerance change related to this PR at all?

Reply:
No, this got mixed in with scipy 1.16 stuff and I missed it when I took that stuff out of this PR.  I'll remove it.